components/identity-mgt/org.wso2.carbon.identity.mgt/src/main/java/org/wso2/carbon/identity/mgt/services/UserInformationRecoveryService.java

```diff
@@ -157,6 +157,18 @@ public VerificationBean verifyUser(String username, CaptchaInfoBean captcha)
     public VerificationBean sendRecoveryNotification(String username, String key, String notificationType)
             throws IdentityMgtServiceException {
 
+        return sendNotification(username, key, notificationType, IdentityMgtConstants.Notification.PASSWORD_RESET_RECOVERY);
```

Comment:
line length should be 120.

Reply:
addressed with https://github.com/wso2/carbon-identity-framework/pull/2409